crates/fj-app/src/main.rs

```diff
@@ -55,13 +58,25 @@ fn main() -> anyhow::Result<()> {
     })?;
     path.push(model);
 
+    let new_error_message = format!("inside default models directory: {}
+                                            \nCan mainly caused by: 
+                                            \n1. Model '{}' can not be found inside '{}' 
+                                            \n2. {} can be mis-typed see inside {} for a match
+                                            \n3. Try to run 'cargo run -- -m cuboid' to know path is connected", 
+                                            abs_path_for_models.display(), 
+                                            path.display(), 
+                                            abs_path_for_models.display(),
+                                            path.display(),
+                                            abs_path_for_models.display(),
+                                        );
```

Comment:
There's a trick to make this kind of thing a bit easier: you can pass each argument only once, and refer to the arguments from the format string by position, so you can use it multiple times. Something like this:

``` rust
format!("{0}, {1}, {0}, {1}, arg0, arg1);
```

This is documented here: https://doc.rust-lang.org/stable/std/fmt/index.html#positional-parameters

Also item 3. in the error message is too specific. It only makes sense within this repository, and won't make sense to anyone who has installed `fj-app` via `cargo install` or by downloading a release binary.

Reply:
@hannobraun yeah, I updated the format and the `abs_path` but I can't figure out how to get the invalid file name? I know its inside the `model` variable everytime if I try to get it, its return back with ` "models/InvalidFile" `

Reply:
Thanks for following up, @ArshErgon! I'll check back and try to answer your question as soon as I can, but I'm on my way out of the house right now. I might not get a chance too look into this until Monday.

Reply:
@hannobraun I figure out, I reformatted the code and added the code.